src/js/utils/can-autoplay.js

```diff
@@ -53,6 +54,10 @@ export function startPlayback (element, { muted }) {
     element.muted = muted;
     element.src = URL.createObjectURL(videoBlob);
 
+    if (OS.tizen) {
+        element.crossOrigin = 'anonymous';
```

Comment:
Nice. Does this work in webkit and chromium? Maybe we should always do this.

Reply:
It's honestly not even working for tizen right now.

Reply:
Maybe check for something new at:
https://github.com/video-dev/can-autoplay
or
https://github.com/Modernizr/Modernizr/blob/master/feature-detects/video/autoplay.js (ignore the ogg check)

Reply:
Maybe this is the issue https://github.com/video-dev/can-autoplay/issues/30